docassemble/GithubFeedbackForm/github_issue.py

```diff
@@ -96,8 +100,13 @@ def feedback_link(
     )
 
 
+def is_likely_spam(body) -> bool:
+    if any([url in body for url in ["boostleadgeneration.com/", "jumboleadmagnet.com/"]]):
+        return True
+    return False
```

Comment:
There was a lot of research into bayesian filters when more people hosted their own mail servers. Should we consider incorporating one? maybe spampy

Reply:
Agreed. This is fine for now, but probably before we add even one more URL to the block list we should try the bayesian classifier approach.